src/win32/wazuh-installer.wxs

```diff
@@ -179,8 +182,13 @@
             <!-- Stop the service as soon as possible on uninstall, and only on uninstall, in order to unlock the files and folders that must be deleted. -->
 			<Custom Action="SetRemoveAllDataValue" Before="InstallFinalize">(NOT UPGRADINGPRODUCTCODE) AND (REMOVE="ALL")</Custom>
  		    <Custom Action="CustomAction_RemoveAllScript" After="SetRemoveAllDataValue">(NOT UPGRADINGPRODUCTCODE) AND (REMOVE="ALL")</Custom>
+ 
+            <Custom Action="SetCustomActionDataValue2" Before="InstallFinalize"/>

```

Comment:
You could add the closing of the `<Custom Action></Custom>`, this would keep the same existing syntax.